packages/ERTP/src/issuer.js

```diff
@@ -10,14 +10,20 @@ import { isPromise } from '@agoric/promise-kit';
 
 import { makeAmountMath, MathKind } from './amountMath';
 import { makeInterface, ERTPKind } from './interfaces';
+import { assertDisplayInfo } from './displayInfo';
 
 import './types';
 
 /**
  * @type {MakeIssuerKit}
  */
-function makeIssuerKit(allegedName, amountMathKind = MathKind.NAT, decimals) {
+function makeIssuerKit(
+  allegedName,
+  amountMathKind = MathKind.NAT,
+  displayInfo = undefined,
+) {
   assert.typeof(allegedName, 'string');
+  assertDisplayInfo(displayInfo);
```

Comment:
There are a number of ways that `allegedDisplayInfo` may misbehave that this code does not protect against. The simplest and safest defense is to *first* make a `pureCopy` using https://github.com/Agoric/agoric-sdk/blob/76e0723915a32b6b76ebea55e4df29d67f8afea8/packages/marshal/src/marshal.js#L30-L32

```suggestion
  displayInfo = pureCopy(displayInfo);
  assertDisplayInfo(displayInfo);
```

Reply:
Done!